storage/swift.go

```diff
@@ -127,6 +127,22 @@ func (s *Swift) Info(ctx context.Context, path string) (types.Info, error) {
 	}, nil
 }
 
+func (s *Swift) Move(ctx context.Context, srcPath, dstPath string) error {
+	return s.conn.ObjectMove(ctx, s.cfg.Container, srcPath, s.cfg.Container, dstPath)
```

Comment:
I prefer we annotate the errors in this lib
```suggestion
	err := s.conn.ObjectMove(ctx, s.cfg.Container, srcPath, s.cfg.Container, dstPath)
	if err != nil {
		return errors.Wrapf(err, "fail to move the object")
	}
	return nil
```